app/views/layouts/application.html.erb

```diff
@@ -192,6 +192,7 @@
 <script src="https://masjumpprdstorage.blob.core.windows.net/responsive-assets/a/webchat-8a47f66c98f9abd3ce2afd710aff5891.js"></script>
 <script type='text/javascript'>
   sWODomain = 'www.moneyadviceservice.org.uk';
+  sWOChatstart = 'https://webchat.moneyadviceservice.org.uk/newchat/chat.aspx';
```

Comment:
thanks, fair point :) I've edited the commit message to reflect the context given in the PR